faker/utils/loading.py

```diff
@@ -4,6 +4,13 @@
 
 def list_module(module):
     path = os.path.dirname(module.__file__)
+    try:
+        import pkgutil
+        modules = [name for finder, name, is_pkg in pkgutil.iter_modules([path]) if is_pkg]
+        if len(modules) > 0:
+            return modules
+    except (ImportError, AttributeError):
+        pass
```

Comment:
I don't think we necessarily care for Python 3.2, but since the fallabck is just one line of code, LGTM